en/tutorials-and-examples/blog/blog.rst

```diff
@@ -130,6 +130,27 @@ SQL statements into your database:
         VALUES ('A title once again', 'And the article body follows.', NOW());
     INSERT INTO articles (title,body,created)
         VALUES ('Title strikes back', 'This is really exciting! Not.', NOW());
+        
+If you are using PostgreSQL, connect to cake_blog database and execute the following SQL instead:
+
+.. code-block:: SQL
+
+   -- First, create our articles table
+   CREATE TABLE articles (
+       id SERIAL PRIMARY KEY,
+       title VARCHAR(50),
+       body TEXT,
+       created TIMESTAMP DEFAULT NULL,
+       modified TIMESTAMP DEFAULT NULL
+   );
+
+   -- Then insert some articles for testing:
+   INSERT INTO articles (title,body,created)
```

Comment:
`INSERT` statements are same for both databases.

Show both `CREATE` statements first, then `INSERT` statements once.

Maybe split first paragraph of this chapter to _we'll just create a single table to store our articles_ and _we'll also throw
in a few articles to use for testing purposes_.